influxql/iterator.gen.go.tmpl

```diff
@@ -673,6 +673,23 @@ func (itr *{{$k.name}}FillIterator) Next() (*{{$k.Name}}Point, error) {
 		switch itr.opt.Fill {
 		case NullFill:
 			p.Nil = true
+		case AverageFill:
```

Comment:
A less conventional way to do this, but probably better, is to just move `AverageFill` above `NullFill` and do this with the condition.

```
{{if or (eq $k.Name "Float") (eq $k.Name "Integer")}}
... code that you have ...
{{else}}
fallthrough
{{end}}
```

That way you don't need `averageString` or `averageBoolean`.